src/frontend/src/lib/components/core/Loader.svelte

```diff
@@ -61,6 +67,12 @@
 
 	let progressModal = false;
 
+	$: {
+		if ($testnets && !$btcAddressTestnet) {
```

Comment:
- `nonNullish`
- should we debounce the loader in case the user have fun toggling off and on the testnet repeatitively?
- while there is no eslint for it, in Oisy we generally `await` promises unless there is a significant reason not too. So I think for consistency reason we can do the same here.

Reply:
- Done!
- Done!
- The is in the top level component where I can't use `await`. I can do a function that does an await inside of the loader, but I won't be able to await that function either. How woudl you do it?

Reply:
> The is in the top level component where I can't use await

Sounds good now with the debounce, we have the same pattern.

You've got a typo though in `debounceLadBtcAddressTestnet` => "Lad", I guess "Load"?

Reply:
Fixed, thanks!